product/models.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+
+class Product(models.Model):
+    name           = models.CharField(max_length = 100)
+    price          = models.DecimalField(max_digits = 10, decimal_places = 2)
+    description    = models.CharField(max_length = 100)
+    main_image_url = models.URLField()
+    category       = models.ManyToManyField('Category', through = 'ProductCategory')
+    item           = models.ManyToManyField('Item', through = 'ProductItem')
+    type           = models.ManyToManyField('Type', through = 'ProductType')
+    Technology     = models.ManyToManyField('Technology', through = 'ProductTechnology')
+    color          = models.ManyToManyField('Color', through = 'ProductColor')
+    size           = models.ManyToManyField('Size', through = 'ProductSize')
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = "products"
+
+class SubImageUrl(models.Model):
+    name = models.URLField()
+    product = models.ForeignKey('Product', on_delete = models.CASCADE)
```

Comment:
```suggestion
class SubImageUrl(models.Model):
    image_url = models.URLField()
    product    = models.ForeignKey('Product', on_delete = models.CASCADE)
```
url은 그 속성명에서부터 url이라는 것을 명시해두시는 것이 좋습니다